canonical/lxd-ui
# feat(test) schedule e2e tests on github

## Done

- schedule e2e tests on PRs
- update images (they were outdated, and the alpine we are running in playwright failed)
- increase timeout for playwright tests (they are sometimes slow in GitHub actions)
- skip vm tests in CI (GitHub actions can only run containers)

Fixes WD-4881

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @lorumic or @edlerd for access.
    - With a local copy of this branch, run as described in the [Readme](https://github.com/canonical/lxd-ui#setting-up-for-development).
2. Perform the following QA steps:
    - check github actions are running the playwright tests
